src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/order/history.yaml

```
@@ -59,13 +59,136 @@ sylius_twig_hooks:
                     addressId: '@=_context.resource.getBillingAddress().getId()'
                     header: 'sylius.ui.billing_address'
                     sort: 'desc'
-                    template: '@SyliusAdmin/order/history/content/sections/addresses/address.html.twig'
+                    template: '@SyliusAdmin/order/history/content/sections/addresses/billing_address.html.twig'
                 priority: 100
             shipping_address:
                 component: sylius_admin:order:address_history
                 props:
                     addressId: '@=_context.resource.getShippingAddress().getId()'
                     header: 'sylius.ui.shipping_address'
                     sort: 'desc'
-                    template: '@SyliusAdmin/order/history/content/sections/addresses/address.html.twig'
-                    priority: 0
+                    template: '@SyliusAdmin/order/history/content/sections/addresses/shipping_address.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.billing_address':
+            header: 
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/header.html.twig'
+                priority: 100
+            details:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.billing_address.header':
+            title:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/header/title.html.twig'
+                priority: 0
+                
+        'sylius_admin.order.history.content.sections.addresses.billing_address.details':
+            modification_status:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/modification_status.html.twig'
+                priority: 100
+            data:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/data.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.billing_address.details.modification_status':
+            label:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/label.html.twig'
+                priority: 100
+            logged:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/logged.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.billing_address.details.data':
+            first_name:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/first_name.html.twig'
```

```suggestion
                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/data/first_name.html.twig'
```
This comment applies to all hookables of this hook